jmxtrans-output/jmxtrans-output-influxdb/src/test/java/com/googlecode/jmxtrans/model/output/InfluxDbWriterTests.java

```diff
@@ -117,7 +119,13 @@ public void pointsAreWrittenToInfluxDb() throws Exception {
 		assertThat(points).hasSize(1);
 
 		Point point = points.get(0);
-		assertThat(point.lineProtocol()).startsWith(lineProtocol);
+		String pointLineProtocol = point.lineProtocol();
+		assertThat(pointLineProtocol).startsWith(lineProtocol);
+		// JMX port is written as field by default :
+		// cf Point.lineProtocol() : there is a space after the tags and one after the fields
+		String[] split = pointLineProtocol.split(" ");
+		assertThat(split).hasSize(3);
+		assertThat(split[1]).contains(JMX_PORT_KEY + "=" + DEFAULT_PORT);
```

Comment:
Yes the position matters : if JMX_PORT_KEY is in the first token (split[0]), that means that it is reported as a tag, whereas if it is in the second token (split[1]), it is reported as a field.
However I'll comment this a little better.